src/main/java/net/atos/zac/flowable/cmmn/ZacCreateHumanTaskInterceptor.java

```diff
@@ -5,10 +5,9 @@
 
 package net.atos.zac.flowable.cmmn;
 
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import static java.util.Collections.unmodifiableMap;
+
+import java.util.*;
```

Comment:
STAR IMPORT is not allowed

Reply:
well, in our Kotlin code conventions it is indeed not allowed but we have no such conventions for the Java code apparently (I thought we did actually but apparently not) so strictly speaking we do allow it. However we have all configured our IntelliJ's so that wildcard imports are not used in Java code so would be good to remove these wildcard imports indeed. Meanwhile I will check if we can change our Java Spotless config to disallow wildcard imports.

Reply:
Changing several imports from the same package into a wildcard import is something that IntelliJ does automatically in my case.